### PR TITLE
Fix localedir

### PR DESCRIPTION
**Pull request by phw**
My suggestion to fix the locale issue.

Disclaimer: This is totally untested as I can't run snapcraft on my system and I won't do any shenanigans necessary to make this run.

If the localedir patch in picard.in works as expected I would consider submitting a PR upstream.

### CODE REVIEW

**Comment by phw** [comment]
Not sure, but gettext itself probably should also be added to build-packages, right?

File: localedir.patch [hunk 1]
```diff
@@ -0,0 +1,11 @@
+diff --git a/scripts/picard.in b/scripts/picard.in
+index 7580da67..18b7f02a 100644
+--- a/scripts/picard.in
++++ b/scripts/picard.in
+@@ -1,3 +1,4 @@
+ #!/usr/bin/env python3
+-from picard.tagger import main; main('%(localedir)s', %(autoupdate)s)
+-
++import os
++localdir = os.path.join(os.environ.get('SNAP', ''), '%(localedir)s')
```
Comment (pachulo) [inline localedir.patch]: Thanks for this! Just one thing: shouldn't it be `localedir = ...`?
```suggestion
+localedir = os.path.join(os.environ.get('SNAP', ''), '%(localedir)s')
```

Comment (phw) [reply localedir.patch]: Thanks, good catch. Fixed

**Comment by phw** [comment]
I hope it works as expected :D

**Pull request merged**